src/checkout/sections/PaymentSection/utils.ts

```diff
@@ -24,14 +24,6 @@ export const getFilteredPaymentGateways = (
 	// we want to use only payment apps, not plugins
 	return compact(paymentGateways).filter(({ id, name }) => {
 		const shouldBeIncluded = supportedPaymentGateways.includes(id);
```

Comment:
Removed this if statement because it was redundant - only the allow-listed gateways will be returned. Also it caused issue with our dummy gateway - its id starts with saleor.io 🙃